src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -323,6 +365,101 @@ export class QuickBar extends LitElement {
     );
   }
 
+  private _generateNavigationCommands(): QuickBarItem[] {
+    const panelItems = this._generateNavigationPanelCommands();
+    const sectionItems = this._generateNavigationConfigSectionCommands();
+
+    return this._withNavigationActions([...panelItems, ...sectionItems]);
+  }
+
+  private _generateNavigationPanelCommands(): Omit<
+    QuickBarNavigationItem,
+    "action"
+  >[] {
+    return Object.keys(this.hass.panels).map((panelKey) => {
+      const panel = this.hass.panels[panelKey];
+
+      return {
+        text: this._getPanelText(panel),
+        icon: "hass:robot",
```

Comment:
This should be updated?

Reply:
Because of the title case thing, you mean?

Reply:
We should use the icon of the panel and not `robot` for every panel?

Reply:
Oh gotcha.

The problem is that `configSection` only sets `iconPath` to the raw SVG data rather than a simple handle.

So either:
1. Half the navigation icons will use some sensible default icon (like an `arrow-right`)
2. I'll have to add the icon handle strings to all configSections. 

My vote would be #2

Reply:
We should add support the svg paths, as that is the way forward for most of our icons.

Reply:
Oh... I jumped the gun and pushed the addition of icon strings :P

So you're saying to instead add `iconPath` to all the `this.hass.panels`? Or is there a helper that converts strings like `hass:sofa` to an SVG path?

Reply:
Fixed issue by supporting `ha-icon` and `ha-svg-icon` in command items